src/ResourceManager/GuestConfiguration/Commands.GuestConfiguration/Cmdlets/GetAzureRmGuestConfigurationPolicyReport.cs

```diff
@@ -0,0 +1,147 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.GuestConfiguration.Cmdlets
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Management.Automation;
+    using Microsoft.Azure.Commands.GuestConfiguration.Common;
+    using Microsoft.Azure.Commands.GuestConfiguration.Models;
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+    using Microsoft.Azure.Management.GuestConfiguration.Models;
+
+    /// <summary>
+    /// Gets InGuest/Vm Policy reports
+    /// </summary>
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "GuestConfigurationPolicyReport", DefaultParameterSetName = ParameterSetNames.InitiativeIdScope), 
```

Comment:
Let's use VMGuestPolicyReport to avoid unnecessary breaking changes.

Reply:
Hi Mark, I discussed in the team. I can change the name to Get-AzureRmVMGuestPolicyReport. However, do you see any issues the name conflicting with Compute cmdlets Get-AzureRmVM* ? Would Compute cmdlet owners be unhappy or users be confused ?

Reply:
changed it to Get-AzureRmVMGuestPolicyReport